### PR TITLE
implemented style-component,enlint error fix, refactor components,X close button using css

### PR DESCRIPTION
**Pull request by eggtart26**

### CODE REVIEW

File: client/src/components/ViewPhotos.jsx [hunk 1]
```diff
@@ -1,20 +1,45 @@
...
-    <div className='ViewPhotosDescription'>Rock House /description go here</div>
-     <h1>Yo it works, this is ViewPhotos.jsx</h1>
+      <button className="leftButton" type="button">left button</button>
+      <button className="rightButton" type="button">right button</button>
+      <button className="exitButton" onClick={props.handleViewButton} type="button">Exit button</button>
+      <img src="https://6-pack.s3-us-west-1.amazonaws.com/00/00.jpg" className="ViewPhotosLarge" />
+
+      <div className="ViewPhotosPage">1/49 /image page go here</div>
+      <div className="ViewPhotosDescription">Rock House /description go here</div>
+      <h1>Yo it works, this is ViewPhotos.jsx</h1>
```
Comment (Maksyym) [inline client/src/components/ViewPhotos.jsx]: Dont forget to remove this whenever you're ready!

File: client/dist/style.css [hunk 2]
```diff
@@ -3,7 +3,7 @@ body {
 }
```
Comment (Maksyym) [inline client/dist/style.css:3]: Just wondering if we are supposed to be using a .css file at all. From my understanding that may cause styling conflicts, but you can always refactor into styled-components/css modules if thats the case!

Comment (eggtart26) [reply client/dist/style.css:3]: yes, thanks for the advice, still working on adding functionality, will do a final reactor after finish all the functionality adding.

**Comment by Maksyym** [comment]
You got it! Work looks great

**Pull request merged**